browser-test/src/admin/admin_predicates.test.ts

```diff
@@ -399,6 +399,65 @@ test.describe('create and edit predicates', () => {
     })
   })
 
+  test('eligibility message field is available to use', async ({
+    page,
+    adminQuestions,
+    adminPrograms,
+    adminPredicates,
+  }) => {
+    await loginAsAdmin(page)
+    await enableFeatureFlag(page, 'customized_eligibility_message_enabled')
+    const programName = 'Test eligibility message field availbale to use'
+    const firstScreen = 'Screen 1'
+    const secondScreen = 'Screen 2'
+
+    await test.step('Adds a program with two screens', async () => {
+      await adminQuestions.addTextQuestion({
+        questionName: 'show-predicate-q',
+        description: 'desc',
+        questionText: 'text question',
+      })
+      await adminQuestions.addTextQuestion({
+        questionName: 'show-other-q',
+        description: 'desc',
+        questionText: 'conditional question',
+      })
+
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.editProgramBlockUsingSpec(programName, {
+        name: firstScreen,
+        description: 'first screen',
+        questions: [{name: 'show-predicate-q'}],
+      })
+      await adminPrograms.addProgramBlockUsingSpec(programName, {
+        name: secondScreen,
+        description: 'screen with predicate',
+        questions: [{name: 'show-other-q'}],
+      })
+    })
+
+    await test.step('Eligibility message field is visible', async () => {
+      await adminPrograms.goToEditBlockEligibilityPredicatePage(
+        programName,
+        firstScreen,
+      )
+      await validateScreenshot(page, 'edit-predicate-eligibility-msg-enabled')
+      await page.getByLabel('Eligibility message').isVisible()
+      await page.getByText('Markdown is supported').isVisible()
+    })
+
+    await test.step('Eligibility message field gets updated', async () => {
+      await adminPredicates.updateEligibilityMessage(
+        'Customized eligibility message',
+      )
+      await validateToastMessage(
+        page,
+        'Eligibility message set to Customized eligibility message',
+      )
+      await validateScreenshot(page, 'edit-predicate-eligibility-msg-updated')
```

Comment:
**suggestion (blocking):** Please narrow the screenshot down to as small of an area as possible, ideally just the eligibility message element.

Reply:
Agree on narrowing the screenshot down to a small of an area, but here imho it's better to show the toast message, which shows up at the top of the page.

Reply:
Ah, I see.  Yes, that makes sense.  Thanks for explaining.